### PR TITLE
Change version to 2.13.999-SNAPSHOT in 2.13 branch

### PR DESCRIPTION
**Pull request by zakkak**
Unblocks https://github.com/graalvm/mandrel/issues/501

Relates to https://github.com/quarkusio/quarkus/pull/30860

### CODE REVIEW

**Comment by gsmet** [comment]
@zakkak have you used the [`update-version.sh`](https://github.com/quarkusio/quarkus/blob/main/update-version.sh) script that is at the root of the repo?

The failures look suspicious.

**Comment by gsmet** [comment]
I can have a look if you prefer.

**Comment by zakkak** [comment]
> @zakkak have you used the [`update-version.sh`](https://github.com/quarkusio/quarkus/blob/main/update-version.sh) script that is at the root of the repo?

No and I guess that's the root cause of the failures. 

> I can have a look if you prefer.

Let me try again with the script this time and if it fails again I guess I could use some help :)

**Comment by zakkak** [comment]
> Let me try again with the script this time and if it fails again I guess I could use some help :)

Still seeing some gradle and TCK failures https://github.com/zakkak/quarkus/actions/runs/5187581104

@gsmet feel free to take over

**Comment by gsmet** [comment]
Closing this one, will try in another PR.
